apt_builder/install_packages.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python
```

Comment:
Nit: maybe rename the file? It doesn't really install anything, it generates a Dockerfile.